src/WebJobs.Script/Scale/TargetScalerVote.cs

```diff
@@ -1,14 +1,14 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Can we just delete the file? If this ever needs to come back, we have source control. Would like to avoid commented out classes in the repo.